email_mailbox/src/utils/electronEventInterface.js

```diff
@@ -1401,6 +1408,31 @@ const handleAddressCreatedEvent = async ({ rowid, params }, accountId) => {
   return { rowid };
 };
 
+const handlePeerSetTrustedEvent = async ({ rowid, params }) => {
+  const { email, trusted } = params;
+  await updateContactByEmail({ email, isTrusted: trusted });
+  const [contact] = await getContactByEmails({ emails: [email] });
+  if (contact) {
+    const contactId = contact.id;
+    emitter.emit(Event.CHANGE_SET_TRUSTED_ACCOUNT, {
+      contactId,
+      isTrusted: trusted
+    });
+  }
+  return { rowid };
+};
+
+const handleBlockRemoteContentEvent = async ({ rowid, params }, accountId) => {
+  // domain not exist in account :O
```

Comment:
?

Reply:
lol, was a comment i put in a moment because i was sorprised that domain is nowhere in account in the database. I suppose that is something that has to be managed at future